internal/command/state_show.go

```diff
@@ -33,13 +33,17 @@ func (c *StateShowCommand) Run(args []string) int {
 	args = c.Meta.process(args)
 	cmdFlags := c.Meta.defaultFlagSet("state show")
 	cmdFlags.StringVar(&c.Meta.statePath, "state", "", "path")
+
+	showSensitive := false
+	cmdFlags.BoolVar(&showSensitive, "show-sensitive", false, "displays sensitive values")
+
 	if err := cmdFlags.Parse(args); err != nil {
 		c.Streams.Eprintf("Error parsing command-line flags: %s\n", err.Error())
 		return 1
 	}
 	args = cmdFlags.Args()
-	if len(args) != 1 {
-		c.Streams.Eprint("Exactly one argument expected.\n")
+	if len(args) == 0 {
+		c.Streams.Eprint("Atleast one argument expected.\n")
```

Comment:
I think that this change is wrong. The arguments have nothing to do with the flags. The one expected argument is the address of a resource, nothing to do with the new flag